docs/src/main/sphinx/release/release-407.md

```diff
@@ -0,0 +1,102 @@
+# Release 407 (14 Feb 2023)
+
+## General
+
+* Add support for correlated queries involving a `VALUES` clause with a single
+  row. ({issue}`15989`)
+* Reduce memory usage for large schemas. This behavior can be enabled with the
+  `query.remote-task.enable-adaptive-request-size` configuration property and
+  configured with the `query.remote-task.max-request-size`,
+  `query.remote-task.request-size-headroom`, and
+  `query.remote-task.guaranteed-splits-per-task` configuration properties or
+  their respective session properties. ({issue}`15721`)
+* Improve concurrency when small, concurrent queries are run on a large cluster
+  by automatically determining how many nodes to use for distributed joins
+  and aggregations. This can be managed with the `query.max-hash-partition-count`
+  (renamed from `query.hash-partition-count`) and `query.min-hash-partition-count`
+  configuration properties. ({issue}`15489`)
+* Improve query memory tracking. ({issue}`15983`)
+* Improve query performance when a predicate evaluates to a null value. ({issue}`15744`)
+* Improve performance of queries with joins on the output of global
+  aggregations. ({issue}`15858`)
+* Improve performance of selective queries, queries that read a small number of
+  columns, and queries that process tables with large Parquet row groups or Orc
+  stripes. ({issue}`15579`)
+* Improve performance of queries with window functions. ({issue}`15994`)
+* Change the `table_type` of materialized views in `information_schema.tables`
+  from `BASE TABLE` to `MATERIALIZED VIEW`. ({issue}`15350`)
+* Fix precision loss when converting `time` values with a precision higher than
+  three and `time with time zone` values with lower precision. ({issue}`15861`)
+* Fix handling of `UNKNOWN` type for row type in exchange operator. ({issue}`15948`)
```

Comment:
UNKNOWN is not a type. If anything, this should say "when processing null values". 

However, the entry should be talking about the user-visible impact. What does "Fix handling" mean? Was there a failure? A performance problem? When does it manifest? What kinds of queries trigger it?

cc @lukasz-stec @sopel39

Reply:
AFAIK, there are no real queries that could trigger this in the current code. I would drop the release note.